config/tsconfig.web.json

```diff
@@ -1,7 +1,7 @@
 {
     "extends": "./tsconfig.base",
     "compilerOptions": {
-        "jsx": "react",
+        "jsx": "react-jsx",
```

Comment:
https://www.typescriptlang.org/tsconfig/#jsx

> - react-jsx: Emit .js files with the JSX changed to _jsx calls optimized for production
> - react: Emit .js files with JSX changed to the equivalent React.createElement calls

https://www.typescriptlang.org/docs/handbook/jsx.html

> "react (classic runtime), react-jsx (automatic runtime)"

The automatic runtime is the new transform (see also ["Manual Babel Setup"](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#how-to-upgrade-to-the-new-jsx-transform))